react-community/react-navigation
# Add React.Element support for TabBar/Drawer and fix docs

Fixes #15 

This pull request fixes docs inconsistencies. It adds missing sections for `tabBar` and `drawer` screen options, clarifies a lot of properties and makes label and icon of tabBar and drawer behave like in header.

Test plan:
- Run NavigationPlayground. There should be no difference.
- Run NavigationPlayground, define either `icon` or a `label`:
```js
({ focused, tintColor }) => <Text>Label</Text>
```
- Run NavigationPlayground, define `label`:
```js
({ focused, tintColor }) => 'Label'
```
or
```
'Label'
```
